int32 bin_id
string size
float64 x_target
float64 y_target
float64 r_target
---
---
